cleanlab_pinecone_rag_workflow/Cleanlab_Pinecone_RAG.ipynb

```diff
@@ -0,0 +1,2673 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "075f77c1-3328-4265-b077-160d496eb048",
+   "metadata": {},
+   "source": [
+    "# How to build a reliable, curated, and accurate RAG system using Cleanlab and Pinecone"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 125,
+   "id": "c70303b2-ce6f-4a03-a460-38bd27e8ae9b",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import pandas as pd\n",
+    "from sentence_transformers import SentenceTransformer\n",
+    "import time\n",
+    "from IPython.display import display\n",
+    "import warnings\n",
+    "import matplotlib.pyplot as plt\n",
+    "import os\n",
+    "import pinecone\n",
+    "import uuid\n",
+    "from pinecone import ServerlessSpec\n",
+    "from typing import List, Tuple, Dict, Optional\n",
+    "\n",
+    "warnings.simplefilter(action='ignore', category=FutureWarning)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 126,
+   "id": "c7ff0b6b-3a2b-4e6a-a41a-b0f7a54e32d2",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "os.environ[\"PINECONE_API_KEY\"] = \"YOUR PINECONE API KEY HERE\""
```

Comment:
Nit but I like 

```python
if not os.environ.get("PINECONE_API_KEY"):
    os.environ[\"PINECONE_API_KEY\"] = \"YOUR PINECONE API KEY HERE\"
```

that way the notebook still runs for folks who have decent creds management 😄